keyboards/handwired/dactyl_manuform/2x7_6_3/keyboard.json

```
@@ -0,0 +1,100 @@
+{
+    "keyboard_name": "Dactyl-Manuform (2x7 + 6 and 3 for thumbs)",
+    "manufacturer": "agia5",
+    "url": "--",
+    "maintainer": "agia5",
+    "usb": {
+        "vid": "0x444D",
+        "pid": "0x3536",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "ws2812": {
+        "pin": "D3"
+    },
+    "matrix_pins": {
+        "cols": ["B4","E6", "D7", "C6", "D4", "D0", "D1"],
+        "rows": ["F4", "F5", "F6", "F7"]
+    },
+    "diode_direction": "COL2ROW",
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D0",
+        "usb_detect": {
```

If you have split usb detect enabled, you might want to add split watchdog support too.
```suggestion
        "transport": {
            "watchdog": true
        },
        "usb_detect": {
```